db/migrate/20191123214158_create_roles.rb

```diff
@@ -6,7 +6,7 @@ def change
 
       t.timestamps
```

Comment:
The migration didn't really change, but it needed to happen after the referee table rename. So I had to delete then rewrite.